src/role-access/containers/specific-access/specific-access-information/specific-access-information.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { ErrorMessage } from '../../../../app/models';
+import { ExclusionNavigationEvent, ExclusionState, ExclusionStateData } from '../../../models';
+import { ExclusionNavigation } from '../../../models/exclusion-navigation.interface';
+import * as fromFeature from '../../../store';
+
+@Component({
+  selector: 'exui-specific-access-information',
+  templateUrl: './specific-access-information.component.html'
+})
+export class SpecificAccessInformationComponent implements OnInit, OnDestroy {
+
+  @Input() public navEvent: ExclusionNavigation;
+  @Input() public title: string = 'Describe the exclusion';
+  @Input() public caption: string = 'Add an exclusion';
+
+  public submitted: boolean = true;
+  public controlName = 'text';
+  public formGroup: FormGroup;
+  public error: ErrorMessage = null;
+  public subscription: Subscription;
+
+  constructor(public readonly store: Store<fromFeature.State>,
+              public readonly fb: FormBuilder) {
```

Comment:
this will cause lint to error I think, change to:

`constructor(
    public readonly store: Store<fromFeature.State>,
    public readonly fb: FormBuilder
)
`

Reply:
No lint error, but I removed the formbuilder as not used.